exercises/convertTemperature/src/listener.js

```diff
@@ -0,0 +1,16 @@
+import handlers from './handlers.js';
+
+import {
+  SPACE,
+  TEMPERATURE_INPUT,
+  CONVERT_TEMPERATURE_CONTAINER,
+  MESSAGE_ERROR_NOT_INTEGER,
```

Comment:
In this file, only the `TEMPERATURE_INPUT` is being used. You can remove the others.

```suggestion
  TEMPERATURE_INPUT,
```